tests/tizen_capi/unittest_tizen_capi.cpp

```diff
@@ -23,6 +23,46 @@ typedef struct
   gboolean playing;
 } TestPipeState;
 
+#if defined (__TIZEN__)
+/**
+ * @brief Test NNStreamer pipeline construct with Tizen cam
+ * @details Failure case to check permission (camera privilege)
```

Comment:
Why do you wanna use @details tag instead of @note or @todo in Doxygen? What for?

Reply:
It was guide of Tizen ACR process